providers/zoho.go

```diff
@@ -53,5 +60,38 @@ func init() {
 				LogoURL: "https://res.cloudinary.com/dycvts6vp/image/upload/v1722471890/media/zoho_1722471890.svg",
 			},
 		},
+		DefaultModule: ZohoCRM,
+		Modules: &Modules{
+			ZohoCRM: {
+				BaseURL:     "https://www.zohoapis.com",
```

Comment:
This needs to be a template value due to multi-region support. See the non-module baseurl

Reply:
We do have customers in EU regions as well as US regions, and they're not at all compatible (as in, tokens generated for one don't work with the other), which is why this matters.